open-api/rest-catalog-open-api.yaml

```diff
@@ -1933,6 +1984,10 @@ components:
           type: array
           items:
             type: string
+        subtype:
```

Comment:
I think it would make more sense to define a new exception type rather than introducing this field

Reply:
Changed to a new exception type.